content/docs/installation/kubectl.md

```diff
@@ -50,17 +50,26 @@ $ kubectl create clusterrolebinding cluster-admin-binding \
 > and potentially broken states. Please ensure you follow the below steps when
 > uninstalling to prevent this happening.
 
-Before continuing, ensure that all cert-manager resources that have been created
+Before continuing, ensure that unwanted cert-manager resources that have been created
 by users have been deleted. You can check for any existing resources with the
 following command:
 
 ```bash
 $ kubectl get Issuers,ClusterIssuers,Certificates,CertificateRequests,Orders,Challenges --all-namespaces
 ```
+It is recommended that you delete all these resources before uninstalling cert-manager. 
+If plan on reinstalling later and don't want to lose some custom resources, you can keep them. 
+However, this can potentially lead to problems with finalizers. Some resources, like 
+`Challenges`, should be deleted to avoid [getting stuck in a pending state](#namespace-stuck-in-terminating-state). 
 
-Once all these resources have been deleted you are ready to uninstall
+Once the unneeded resources have been deleted, you are ready to uninstall
```

Comment:
Same as above- if we assume that CRDs are deleted there is no way to keep custom resources. If CRDs are managed separately there will be no need to delete custom resources.